src/themes/default/components/core/blocks/Microcart/Product.vue

```diff
@@ -204,6 +217,18 @@ export default {
     },
     productQty () {
       return this.editMode ? this.getEditingQty : this.product.qty
+    },
+    isSimpleOrConfigurable () {
+      if (
+        this.product.type_id === 'simple' ||
+        this.product.type_id === 'configurable'
+      ) { return true }
+      return false
```

Comment:
```suggestion
      return ['simple', 'configurable].includes(this.product.type_id)
```

Reply:
thanks :) also applied this for `src/themes/default/pages/Product.vue`